### PR TITLE
[platform.hashing/0.3.0] Add requirements

### PR DESCRIPTION
**Pull request by FreePhoenix888**
Specify library name and version:  **platform.hashing/0.3.0**

---

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Review by uilianries — DISMISSED** [review]

File: recipes/platform.hashing/all/conanfile.py [hunk 1]
```diff
@@ -47,6 +47,10 @@ def _compilers_minimum_version(self):
     def _minimum_cpp_standard(self):
         return 20
 
+    def requirements(self):
+    		if Version(self.version) >= "0.3.0":
```
Comment (prince-chrismc) [inline recipes/platform.hashing/all/conanfile.py:51]: Why did https://github.com/conan-io/conan-center-index/pull/11490 pass?

Comment (FreePhoenix888) [reply recipes/platform.hashing/all/conanfile.py:51]: I have the same question @prince-chrismc 🌚

Comment (uilianries) [reply recipes/platform.hashing/all/conanfile.py:51]: I don't know, but please, do not mix space and tabs. Usually, python is sensitive to it.

**Review by uilianries — APPROVED** [review]

**Review by jwillikers — APPROVED** [review]

**Review by prince-chrismc — APPROVED** [review]

File: recipes/platform.hashing/all/conanfile.py (hunk 1 above)
Comment (prince-chrismc) [reply recipes/platform.hashing/all/conanfile.py]: nothing obvious in the code shows where this is being used 🙈 https://github.com/linksplatform/Hashing/compare/0.2.0...cpp_0.3.0

maybe the test package needs a little help

**Pull request merged**